provisioning/templates-api/aws.sh

```diff
@@ -15,7 +15,16 @@ fi
 # Get credentials to the Amazon Elastic Kubernetes Service
 aws eks update-kubeconfig --name "$AWS_EKS_CLUSTER_NAME" --region "$AWS_REGION"
 
+if [ "${TMP_ETCD_REDEPLOY:-false}" = "true" ]
+then
+    kubectl delete deployment templates-api -n templates-api
```

Comment:
Tempaltes API je v produkcii, nemoze sa len tak vypnut.
Nevadi mu ked sa vypne etcd, to sa pouziva iba na zamky a je to schvalne odolne proti vypadkom etcd, kedze to bezi iba v jednej instancii.

Reply:
Takze toto sa musi odstranit zo scriptu, tento riadok.

Reply:
Jo tím jsme si nebyl jistý, viz co jsem psal na slack. Odeberu.